src/main/Java/AnagramKata.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class AnagramKata {
+    public static Scanner s;
+
+    public AnagramKata(String path) throws FileNotFoundException {
+        File wordList = new File(path);
```

Comment:
We spoke briefly about the issues with reading from explicit file paths - see [here](https://www.baeldung.com/reading-file-in-java#1-using-standard-java) for an explanation of how to read files from the classpath using standard Java.